server/data/data-jmap-cassandra/src/main/java/org/apache/james/jmap/cassandra/vacation/CassandraVacationDAO.java

```diff
@@ -104,6 +92,29 @@ public CassandraVacationDAO(Session session, CassandraTypesProvider cassandraTyp
                 .getZonedDateTime());
     }
 
+    private Insert createSpecificUpdate(VacationUpdate vacationUpdate, Insert baseInsert) {
+        Insert insertWithSubject = addValue(baseInsert, CassandraVacationTable.SUBJECT, vacationUpdate.getSubject());
```

Comment:
More clear, thank you.
To reduce well without super type ackwardness, it have to be Function<Insert, Insert> and not UnaryOperator<Insert>.
Otherwise, it remove order from the pipeline to generate Insert. Cool :-)